src/lib/IconBase.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { forwardRef, useContext } from "react";
+import { IconProps, IconContext } from "../lib";
+
+const IconBase = forwardRef<SVGSVGElement, IconProps>((props, ref) => {
+  const { color, size, weight, mirrored, children, renderPath, ...restProps } = props;
+
+  const {
+    color: contextColor,
+    size: contextSize,
+    weight: contextWeight,
+    mirrored: contextMirrored,
+    ...restContext
+  } = useContext(IconContext);
+
+  return (
+    <svg
+      ref={ref}
+      xmlns="http://www.w3.org/2000/svg"
+      width={size ?? contextSize}
+      height={size ?? contextSize}
+      fill={color ?? contextColor}
+      viewBox="0 0 256 256"
+      transform={mirrored || contextMirrored ? "scale(-1, 1)" : undefined}
+      {...restContext}
+      {...restProps}
+    >
+      {children}
```

Comment:
`children` should go above the bounding box, but I LOVE this.

Reply:
I think I don't understand where to put the children. I put it in the same place where is in all the other icons.

<img width="592" alt="Screenshot 2021-03-08 at 19 35 48" src="https://user-images.githubusercontent.com/52796523/110372015-aab4b380-8045-11eb-97f9-1a1bbb7d8029.png">

Reply:
Sorry, you're 100% right! _I_ misunderstood what your IconBase was doing. It takes a render prop, not children, so we're all good! In this context, `children` are svg-namespaced elements you can add to an icon to modify them, e.g. adding a background shape or [animating](https://github.com/phosphor-icons/phosphor-react#composability).